### ISSUE TITLE
0995 - 508 audit defect 7

### ISSUE THREAD

**Issue by bacitracin** [department-of-veterans-affairs/va.gov-team#109234]
## Issue

This ticket is for tracking fixes for Defect 7 from the recent 508 audit on the 0995. Defect 7 is super broad and will require fixes to multiple design system components.

> Throughout the product, when the screen is initially loaded, the names for the radio buttons are read once when reading with the arrow keys but after selecting a radio button, the names are read twice by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen, when reading with the arrow keys, “Disability compensation claim radio button not checked” is read but after selecting this option, “Disability compensation claim radio button checked Disability compensation claim” is read with the arrow keys. (New: 04/04/2025)
> 
> Throughout the product, after expanding some buttons, the content cannot be read by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen (Follow these steps to get started), after expanding What happens after I apply?, the content cannot be read by assistive technology. Similarly, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 4Veteran information), after expanding Are you experiencing domestic violence?, some of the content cannot be read. (New: 04/04/2025)
> 
> Throughout the product, some elements are not read correctly by assistive technology. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 5Veteran information), under Personal information, when tab is used to move the focus to the Find more detailed instructions for how to change your legal name … link, “Find more detailed instructions for how to change your legal name on file (opens in a new tab)opens in a new tab” is read by assistive technology—“opens in a new tab” is read twice. Similarly, on the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), “Review the notice of evidence needed (opens in a new tab)opens in a new tab” is read. (New: 04/04/2025) - Should be fixed, doublecheck
> 
> Throughout the product, the names for check boxes are read twice when reading with the arrow keys. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step1 of 5Veteran information), “I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground. check box not checked I live or sleep overnight in a place that isn’t meant for regular sleeping. This includes a car, park, abandoned building, bus station, train station, airport, or camping ground.” is read. (New: 04/04/2025)
> 
> Throughout the product, extra information read after the role does not convey correct information. For example, on the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), when tab is used to move the focus to the check box, “I certify that I have reviewed the notice of evidence needed or I received my decision less than 1 year ago. (*Required) check box not checked If you’re filing a Supplemental Claim more than 1 year after you got your decision notice, you’ll need to review and acknowledge our notice of evidence needed for your disability claim. Review the notice of evidence needed (opens in a new tab) opens in a new tab” is read—this does not open a new tab. On the File A Supplemental Claim | Veterans Affairs screen (Step3 of 5New and relevant evidence), when tab is used to move the focus to the I acknowledge and authorize this release of information (*Required) check box, “… this authorization is signed in Item 11, as w” is read—the text is too long to be read. (New: 04/04/2025)

## Resources
[[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms](https://github.com/department-of-veterans-affairs/va.gov-team/issues/106900#top)
[[ADE 508 Audit]: 508 Audit Report for 43785 va.gov Forms - Defect 7 | High #107044](https://github.com/department-of-veterans-affairs/va.gov-team/issues/107044)
